observability/grafana/AIBrix_Envoy_Gateway_Plugins_Dashboard.json

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
Great job! Do you have a screenshot of this dashboard? Feel free to attach one in the PR

Reply:
changes moved to https://github.com/vllm-project/aibrix/pull/1211
will move your comment there and discuss / address